src/assets/locales/resources_en_US.json

```diff
@@ -31,6 +31,22 @@
     "_description": "Label for the repeater item checkbox for the repeater component",
     "message": "Select row"
   },
+  "skyux_repeater_item_reorder_instructions": {
+    "_description": "Accessible operation instructions for reordering a repeater item",
+    "message": "Item grabbed. Press up and down arrow keys to change position, spacebar or enter to drop, escape key to cancel."
+  },
+  "skyux_repeater_item_reorder_moved": {
+    "_description": "Alerts the user to the new position an items moved to. Does not include variable due to usage case",
+    "message": "Moved item to position"
+  },
+  "skyux_repeater_item_reorder_operation": {
+    "_description": "Accessible operation instructions for triggering reordering a repeater item",
+    "message": "Press spacebar or enter to reorder"
+  },
+  "skyux_repeater_item_reorder_top": {
+    "_description": "Accessible label for button to reorder a repeart item to the top of the list",
```

Comment:
Not sure if it matters, but I'd change `repeart` to `repeater`

Reply:
Done